sfccdevops/sfcc-cli
# Feature - Live Reload

#### What's this PR do?

Adds "Remote Control" for Sandbox Storefront via `sfcc remote`

#### Where should the reviewer start?

Mainly just need to check out the code and see if there is anything super obvious that needs fixed.

#### How should this be manually tested?

* `./bin/cli.js watch` in one terminal window
* `./bin/cli.js remote` in another terminal window
* make sure script mentioned in `./docs/cmd-remote.md` is on your sandbox
* edit some code
* make sure page reloads

#### Any background context you want to provide?

This feature is mainly to help speed up development and prevent switching back and forth from the developers IDE, to their browser, moving their mouse to click the reload button ( or just giving the window focus ) then switching back to the IDE.

#### Screenshots (if appropriate)

![remote](https://user-images.githubusercontent.com/508411/51102265-428f1d80-17ac-11e9-82b3-b28c733ece37.gif)

#### What gif best describes this PR or how it makes you feel?

![80s_dancing](https://user-images.githubusercontent.com/508411/51102370-b16c7680-17ac-11e9-94ce-543c56cb126a.gif)

#### Definition of Done:
